### PR TITLE
Update setup-osx.rb

### PR DESCRIPTION
**Pull request by jweakley**

### CODE REVIEW

File: development/setup-osx.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def self.full_install
   def ensure_bash_profile
     bash_profile_location = '~/.bash_profile'
     unless File.exist?(bash_profile_location)
-      File.open(bash_profile_location, 'w') {}
+      File.write(File.expand_path(bash_profile_location), "")
```
Comment (chadzink) [inline development/setup-osx.rb]: You can just write to a file without opening it?

Comment (Samsinite) [inline development/setup-osx.rb]: It's a wrapper around opening, writing the string to the file, and closing it.

Comment (jweakley) [inline development/setup-osx.rb]: Was this the only feed back you both had on this?

**Comment by Samsinite** [comment]
:+1:

**Pull request merged**